Automattic/wp-calypso
# Jetpack Backup: Add backup warnings list

#### Proposed Changes

* Adds a list of warnings to the daily backup if warnings exist
* The warning category messages are currently being reviewed

#### Testing Instructions



* Have the backup team give you access to a site with warnings
* Visit the Calypso live links below and access those sites to view the changes
* Review a site without warnings to ensure no warning messaging appears (or other issues)

<img width="738" alt="Screen Shot 2022-06-02 at 4 39 57 PM" src="https://user-images.githubusercontent.com/1992658/171734224-d05a3258-6e18-4245-962d-697d339cdea6.png">
